components/CollaboratorsAmount/SubmitForm.tsx

```diff
@@ -7,23 +8,41 @@ const SubmitForm = () => {
   const { setTab, collaboratorsAmount, setCollaboratorsAmount } =
     useContractBuilderProvider()
 
+  const handleCollaboratorsChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value
+
+    if (value === '') {
+      setCollaboratorsAmount(0)
+
+      return
+    }
```

Comment:
What happens if you `remove` this `if` statement???
- what value is it adding that is not already captured by your `isNaN` check???

Reply:
it is tackling when the input filed is empty